index.js

```diff
@@ -72,7 +72,8 @@ function beautify(error, collection, values, messages) {
                 indexes[indexName].forEach(function (field) {
                     var path = field[0];
                     var props = {
-                        type: 'Duplicate value',
+                        type: 'unique',
+                        kind: 'unique',
```

Comment:
I think this is why: https://github.com/matteodelabre/mongoose-beautiful-unique-validation/pull/40#discussion_r131784417 — still, we shouldn’t keep the duplicated attribute IMHO.